doc/user/content/sql/create-connection.md

```diff
@@ -149,23 +149,23 @@ Field                       | Value            | Required | Description
 `PORT`                      | `int4`           | ✓        | Port for the connection.
 `USER`                      | `text`           | ✓        | Username for the connection.
 
-#### Public key retrieval
-
-Once a connection is created, you can find and retrieve the public keys to be used in the bastion server by running:
+##### Example
 
-`SELECT * FROM mz_ssh_tunnel_connections;`
+```sql
+    CREATE CONNECTION ssh_connection TO SSH TUNNEL (
+        HOST '<SSH_BASTION_HOST>',
+        USER '<SSH_BASTION_USER>',
+        PORT 1
```

Comment:
Same here: to avoid user confusion we could change 1 to `<SSH_BASTION_PORT>`,

Reply:
Agree!